types/elo-rank/index.d.ts

```diff
@@ -0,0 +1,12 @@
+// Type definitions for elo-rank 1.0
+// Project: https://github.com/dmamills/elo-rank#readme
+// Definitions by: Antonio Pavlinovic <https://github.com/apavlinovic>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export default class EloRank {
```

Comment:
It's not `export default`: https://github.com/dmamills/elo-rank/blob/master/index.js#L23